dockstore-webservice/src/main/java/io/dockstore/webservice/resources/UserResource.java

```diff
@@ -465,4 +465,36 @@ public ArrayList getOrganizations(@ApiParam(hidden = true) @Auth User authUser)
         }
         return null;
     }
+    @GET
+    @Timed
+    @UnitOfWork
+    @Path("/{userId}/bitbucketUser")
+    @ApiOperation(value = "Test bitbucket", notes = "NO authentication", response = String.class)
```

Comment:
This was implemented by Victor I think, I just changed it to grab user information based on oauth information.  It could be useful but it is equivalent to calling the bitbucket api so I think it doesn't really have a useful purpose.  I'll remove it and regenerate swagger.